projects/robots/telerob/telemax_pro/controllers/telemax_demo/telemax_demo.c

```diff
@@ -24,10 +24,10 @@
 
 #define TIME_STEP 32
 
-void noop(int n_steps) {
-  // Wait without applying commands during 'n_steps' steps.
+void noop(int duration) {
+  // Wait without applying any command during 'duration' simulation steps.
```

Comment:
```suggestion
  // Wait without applying any command during 'duration' milliseconds.
```

Reply:
Your suggestion is wrong, the unit of the argument is a number of simulation steps, not milliseconds. Please check the function content.

Reply:
Therefore `n_steps` is better than `duration` IMO

Reply:
I don't think so: `counter` is accumulating `TIME_STEP` which is expressed in milliseconds and then compared to `duration` which should therefore be milliseconds as well.

Reply:
Oops, you're right. Sorry for this.